ansible/workers.yml

```diff
@@ -134,3 +134,27 @@
       - python3-gobject
       state: installed
     when: ansible_distribution == 'Fedora'
+
+  - name: Ensure OSCAP Anaconda dependencies are installed (RHEL7 only)
+    package:
+      name:
+      - anaconda
+      - python2-mock
+      - python-nose
+      - python-cpio
+      - zanata-python-client
+      state: installed
+    when: ansible_distribution == 'RedHat' or ansible_distribution_version.split(".")[0] >= "7"
+
+  - name: Ensure OpenSCAP Anaconda dependencies are installed (Fedora only)
+    package:
+      name:
+      - anaconda
+      - python3-mock
+      - python3-nose
+      - python3-cpio
+      - python3-kickstart
+      - zanata-python-client
```

Comment:
Can you add `python3-pycurl` ? It is required as well. See

https://jenkins.open-scap.org/job/oscap-anaconda-addon-pull-requests/distributions=fedora/10/console

[Note, I already installed it on fedora machine - now it fails on something else again...]
https://jenkins.open-scap.org/job/oscap-anaconda-addon-pull-requests/distributions=fedora/11/console

But that is not in the scope of Jenkins configuration.